### PR TITLE
Update KafkaClusterRequirement.md 

### PR DESCRIPTION
**Pull request by ravisagar100**
KafkaClusterRequirement.md - updated with external birnami kafka example under TASK-1364387 in US-506379

### CODE REVIEW

**Comment by ravisagar100** [comment]
@taz-pega-work  FYI&A

**Comment by ravisagar100** [comment]
@taz-pega-work  we have decided to move this document update in [KafkaClusterRequirement.md] please approve and we dont need workflow approval as this is only documentation

File: charts/pega/KafkaClusterRequirement.md
Comment (taz-pega-work) [inline charts/pega/KafkaClusterRequirement.md:55]: Can you tweak this before you merge so it's proper English??

For testing purpose, deploy a bitnami kafka cluster using the provided Helm command example where `my-kafka` is the namespace of your Kafka service deployment.

**Review by taz-pega-work — APPROVED** [review]
looks good but I'd make it clear about what you command is doing so clients trust the example

File: charts/pega/KafkaClusterRequirement.md
Comment (ravisagar100) [reply charts/pega/KafkaClusterRequirement.md:55]: you mean add the below sentence? is it now not approved? I cant see the changes yet.. please let me know if I should resubmit the PR

For testing purpose, deploy a bitnami kafka cluster using the provided Helm command example where my-kafka is the namespace of your Kafka service deployment.

Comment (taz-pega-work) [reply charts/pega/KafkaClusterRequirement.md:55]: I think it is useful to add that sentence, but I'm not going to block your work. I'm just not going to let improper English slip by during a review, since we should uphold a standard. It's your choice, not mine. I approved the PR.

File: charts/pega/KafkaClusterRequirement.md
Comment (ravisagar100) [inline charts/pega/KafkaClusterRequirement.md:56]: For testing purpose, deploy a bitnami kafka cluster using the provided Helm command example where my-kafka is the namespace of your Kafka service deployment.

Comment (ravisagar100) [reply charts/pega/KafkaClusterRequirement.md:56]: @taz-pega-work  have done the changes suggested

File: charts/pega/KafkaClusterRequirement.md
Comment (ravisagar100) [reply charts/pega/KafkaClusterRequirement.md]: Hi @taz-pega-work  please approve the changes

**Comment by ravisagar100** [comment]
@taz-pega-work  please proceed with review and approval.

**Review by taz-pega-work — APPROVED** [review]
looks great. thank you!

**Review by kishorv10 — APPROVED** [review]

**Pull request merged**
